### PR TITLE
Bigtable Write samples 

### PR DESCRIPTION
**Pull request by billyjacobson**

### CODE REVIEW

File: bigtable/snippets/src/main/java/com/example/bigtable/WriteConditionally.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  public static void writeConditionally(String projectId, String instanceId, String tableId) {
+    // String projectId = "my-project-id";
+    // String instanceId = "my-instance-id";
+    // String tableId = "mobile-time-series";
+
+    try (BigtableDataClient dataClient = BigtableDataClient.create(projectId, instanceId)) {
+      long timestamp = System.currentTimeMillis() * 1000;
+
+      String rowKey = "phone#4c410523#20190501";
```
Comment (lesv) [inline bigtable/snippets/src/main/java/com/example/bigtable/WriteConditionally.java]: dynamic UUID?

File: bigtable/snippets/src/main/java/com/example/bigtable/WriteIncrement.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  public static void writeIncrement(String projectId, String instanceId, String tableId) {
+    // String projectId = "my-project-id";
+    // String instanceId = "my-instance-id";
+    // String tableId = "mobile-time-series";
+
+    try (BigtableDataClient dataClient = BigtableDataClient.create(projectId, instanceId)) {
+      // Get an existing row that has a cell with an incrementable value. A value can be incremented
+      // if it is encoded as a 64-bit big-endian signed integer.
+      String rowKey = "phone#4c410523#20190501";
```
Comment (lesv) [inline bigtable/snippets/src/main/java/com/example/bigtable/WriteIncrement.java]: Dynamic UUID?

File: bigtable/snippets/src/main/java/com/example/bigtable/WriteSimple.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  public static void writeSimple(String projectId, String instanceId, String tableId) {
+    // String projectId = "my-project-id";
+    // String instanceId = "my-instance-id";
+    // String tableId = "mobile-time-series";
+
+    try (BigtableDataClient dataClient = BigtableDataClient.create(projectId, instanceId)) {
+      long timestamp = System.currentTimeMillis() * 1000;
+
+      String rowKey = "phone#4c410523#20190501";
```
Comment (lesv) [inline bigtable/snippets/src/main/java/com/example/bigtable/WriteSimple.java]: dynamic / random UUID?

**Comment by billyjacobson** [comment]
> LGTM - I generally like this, asked a few Q's, ok to ignore me.
> Do the docs talk a bit about key generation? Should the comments in the code?

The docs will talk about how the row key is created. Could you take a look at the way I am writing bytes in the write simple and write batch files? It works, but I'm not sure if there is a cleaner way to do this

File: bigtable/hbase/snippets/src/main/java/com/example/bigtable/WriteBatch.java [hunk 4]
```diff
@@ -38,24 +38,21 @@ public static void writeBatch(String projectId, String instanceId, String tableI
...
+      byte[] one = new byte[] {0, 0, 0, 0, 0, 0, 0, 1};
 
       List<Put> puts = new ArrayList<Put>();
       puts.add(new Put(Bytes.toBytes("tablet#a0b81f74#20190501")));
       puts.add(new Put(Bytes.toBytes("tablet#a0b81f74#20190502")));
 
-      puts.get(0)
-          .addColumn(
-              COLUMN_FAMILY_NAME, Bytes.toBytes("connected_wifi"), timestamp, Bytes.toBytes(1));
+      puts.get(0).addColumn(COLUMN_FAMILY_NAME, Bytes.toBytes("connected_wifi"), timestamp, one);
```
Comment (lesv) [inline bigtable/hbase/snippets/src/main/java/com/example/bigtable/WriteBatch.java]: Do you want to comment what's happening here?

**Comment by lesv** [comment]
I need to read a bit, will give more feedback in a few hours.

**Comment by lesv** [comment]
The only thing I'd recommend is comments explaining what's happening, and why your qualifiers all have different stuff each time you use it.  I count 4 different ways of specify it.

**Pull request merged**